projects/core/src/auth/user-auth/services/oauth-lib-wrapper.service.ts

```diff
@@ -16,6 +16,11 @@ import { AuthConfigService } from './auth-config.service';
 export class OAuthLibWrapperService {
   events$: Observable<OAuthEvent> = this.oAuthService.events;
 
+  /**
+   * Returns true if the `tryLogin()` call has not completed.
+   */
+  loginInProgress$: Observable<boolean> = new BehaviorSubject<boolean>(false);
```

Comment:
You needn't define an explicit type of property if it's easily inferred by TS. Also calling a method with false causes the type (boolean) to be easily pressed.
![image](https://user-images.githubusercontent.com/8040837/165411069-4833767d-c8e4-4bb3-a55f-80f2cce33332.png)

In this case, you tricked TS that it's an `Observable` by unnecessary type narrowing, making you unable to explicitly refer to the `next` method (which occurs in `BehaviorSubject`, but not in `Observable`) if you have to use an assertion somewhere, there is most likely an oversight somewhere.